### PR TITLE
routing the future

### PR DESCRIPTION
**Pull request by saketh-are**
@gushchinav @edufgf

### CODE REVIEW

**Review by edufgf — CHANGES_REQUESTED** [review]
Sorry but I already have patents over this.
:rabbit2:

File: sqrt.cpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+namespace nithin_michael {
```
Comment (gushchinav) [inline sqrt.cpp:3]: Where are the unit tests???

Comment (saketh-are) [reply sqrt.cpp:3]: How do I ban you from my repository?

**Review by edufgf — CHANGES_REQUESTED** [review]
pop

**Pull request closed without merging**